homeassistant/components/axion_dmx/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "axion_dmx",
```

Comment:
DMX (Digital Multiplex) is a protocol used to control devices such as lights or fog machines. The signal is unidirectional, meaning it only travels in one direction; from the controller or first light, all the way to the last. It can control up to 512 channels of data per universe, where each channel typically represents one parameter (such as brightness or color).

In its most basic form, DMX is just a protocol for lights. 

Here's the link to the protocol: https://tsp.esta.org/tsp/documents/docs/ANSI-ESTA_E1-11_2008R2018.pdf